benchmarks/benchmark/tools/locust-load-inference/locust-docker/locust-tasks/tasks.py

```diff
@@ -99,39 +100,43 @@ def generate_request(prompt):
         raise ValueError(f"Unknown backend: {backend}")
     return pload
 
+
 def get_token_count(prompt, resp):
     """Get number of tokens to prompt and resp using the tokenizer"""
     global tokenizer
     backend = model_params["backend"]
-    
+
     number_of_input_tokens = len(tokenizer.encode(prompt))
     number_of_output_tokens = 0
-    
+
     if backend == "vllm":
-        number_of_output_tokens = 0 # to be added
+        number_of_output_tokens = 0  # to be added
     elif backend == "tgi":
-        number_of_output_tokens = 0 # to be added
+        number_of_output_tokens = 0  # to be added
     elif backend == "tensorrt_llm_triton":
         resp_dict = json.loads(resp.content.decode('utf-8'))
-        number_of_output_tokens = len(tokenizer.encode(resp_dict['text_output']))
+        number_of_output_tokens = len(
+            tokenizer.encode(resp_dict['text_output']))
     elif backend == "sax":
-        number_of_output_tokens = 0 # to be added
+        number_of_output_tokens = 0  # to be added
     else:
         raise ValueError(f"Unknown backend: {backend}")
     return number_of_input_tokens, number_of_output_tokens
 
 
-
 class BenchmarkUser(FastHttpUser):
     weight = 1
+    # Connection_timeout default is 60s. For inferencing workloads with a large payload
+    # this timeout can be too short. Increasing timeout to large amount.
+    # TODO: turn timeout into a variable.
```

Comment:
is this used anywhere? or is it just a placeholder for that TODO?

Reply:
placeholder for a todo :) no tracking issue at this point